.github/workflows/deploy.yml

```diff
@@ -36,49 +37,35 @@ jobs:
           docker_context: mediator # optional, default .
           identity_provider: ${{ secrets.NAIS_WORKLOAD_IDENTITY_PROVIDER }}
           project_id: ${{ vars.NAIS_MANAGEMENT_PROJECT_ID }}
-      - name: Archive NAIS yaml
-        if: github.ref == 'refs/heads/main'
-        uses: actions/upload-artifact@v3
-        with:
-          name: nais-yaml
-          path: mediator/nais
+
     outputs:
       image: ${{ steps.docker-build-push.outputs.image }}
 
   deploy-dev:
     needs: [ build ]
-    if: github.ref == 'refs/heads/main'
```

Comment:
var det meninga å fjerne denna @androa ?

Reply:
Jaa, det var jo på en måte det. For å få testa i dev. Men kan legge den inn igjen nå når det er merga.